src/osm/GeometryHandler.cpp

```diff
@@ -58,6 +58,7 @@ using osm2rdf::ttl::constants::IRI__OPENGIS_INTERSECTS;
 using osm2rdf::ttl::constants::IRI__OSM2RDF_CONTAINS_AREA;
 using osm2rdf::ttl::constants::IRI__OSM2RDF_CONTAINS_NON_AREA;
 using osm2rdf::ttl::constants::IRI__OSM2RDF_INTERSECTS_AREA;
+
```

Comment:
I don't think we need this empty line here.

Reply:
Indeed, I removed ith with ba2e1f6